packages/contentful--create-contentful-app/src/app-actions/javascript/build-actions.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = fs;
+
+const buildActions = () => {
+  mkdirSync('build/actions', { recursive: true });
```

Comment:
Would be great to console.log this creation operation and success/fail messages. Plus wrap it in try..catch.